yamlparse-applicative/src/YamlParse/Applicative/Pretty.hs

```diff
@@ -92,17 +111,21 @@ schemaDoc = go emptyComments
             ArraySchema t s -> "-" <+> align (go (addMComment cs t) s)
             -- The comments really only work on the object level
             -- so they are erased when going down
-            ObjectSchema t s -> e (ge s) (addMComment cs t)
+            ObjectSchema t s -> vsep
+              [ e "<object>" (addMComment cs t)
```

Comment:
Why did you add this piece? 
Now it looks like there are two pieces of an object expected:
![object](https://user-images.githubusercontent.com/3521180/82754099-05d52c80-9dcb-11ea-8ba5-2d8697d7f999.png)

Reply:
In the top-level case it looks weird (although correct I think?) but if you check a nested object, I think that's an improvement. If you check `Data.Yaml.Value`, this is the correct YAML data type.

Reply:
Ah yes, your'e right. I'll add a special case for the top level like this and revert the normal case:
```
            ObjectSchema t AnySchema -> e "<object>" (addMComment cs t)
            ObjectSchema t s -> ge s
```